src/llama-model.cpp

```diff
@@ -16021,17 +16020,16 @@ struct llm_build_plamo2 : public llm_graph_context_mamba {
 
             Kcur = build_norm(Kcur, model.layers[il].attn_k_norm, NULL, LLM_NORM_RMS, il);
             cb(Kcur, "Kcur_normed", il);
-
             Kcur = ggml_rope_ext(
                     ctx0, Kcur, inp_pos, nullptr,
                     n_rot, rope_type, n_ctx_orig, freq_base, freq_scale,
                     ext_factor, attn_factor, beta_fast, beta_slow
                     );
 
-            cur = build_attn(inp, model.layers[il].wo, NULL, Qcur, Kcur, Vcur, NULL, NULL, 1.0f, il);
+            cur = build_attn(inp, model.layers[il].wo, NULL, Qcur, Kcur, Vcur, NULL, NULL, 1.0f/sqrtf(float(n_embd_head_v)), il);
         }
 
-        cb(cur, "attn_out", il);
+        cb(cur, "attn_output", il);
```

Comment:
```suggestion
        cb(cur, "attn_out", il);
```
Not that it really matters, but any reason to change this? `attn_out` is the most commonly used name.

Reply:
Thanks, it was unnecessary change, so should be reverted.